app/charts/column/columns-grouped.tsx

```diff
@@ -57,16 +58,43 @@ export const ErrorWhiskers = () => {
 
   React.useEffect(() => {
     if (ref.current && renderData) {
+      const renderWhiskers = (
+        g: Selection<SVGGElement, null, SVGGElement, unknown>
+      ) => {
+        g.selectAll<SVGGElement, RenderWhiskerDatum>("g")
+          .data(renderData, (d) => d.key)
+          .call(renderWhisker);
+      };
+
       select(ref.current)
-        .selectAll<SVGGElement, RenderWhiskerDatum>("g")
-        .data(renderData, (d) => d.key)
-        .call(renderWhisker);
+        .selectAll<SVGGElement, null>(".content")
+        .data([null])
```

Comment:
For my information, why do we put a null here ?

Reply:
I used the `join` pattern here to have access to every lifecycle event of the element (and not to animate the `transform` property initially) – because we are using `selectAll`, and the `[null]` is an 1-element array that makes the logic render one element. It could be anything, but as the data in this context doesn't matter, it's a convention (I think?) to use `null` :)